GramAddict/__init__.py

```diff
@@ -178,15 +179,12 @@ def run():
                 "Could not get one of the following from your profile: username, # of followers, # of followings. This is typically due to a soft ban. Review the crash screenshot to see if this is the case."
             )
             logger.critical(
-                f"Username: {getattr(session_state,'my_username')}, Followers: {getattr(session_state,'my_followers_count')}, Following: {getattr(session_state,'my_following_count')}"
+                f"Username: {session_state.my_username}, Followers: {session_state.my_followers_count}, Following: {session_state.my_following_count}"
             )
             save_crash(device)
             exit(1)
 
-        username = session_state.my_username
-        followers = session_state.my_followers_count
-        following = session_state.my_following_count
```

Comment:
I don't think I did this, but it was silly to set these as a variable and use them once.